app/src/main/dist/bin/crate-keystore

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+source "`dirname "$0"`"/crate-env
+
+exec \
+  "$JAVA" \
+  $CRATE_JAVA_OPTS \
+  -Des.path.home="$CRATE_HOME" \
+  -Des.path.conf="$PATH_CONF" \
+  -cp "$CRATE_CLASSPATH" \
+  org.elasticsearch.common.settings.KeyStoreCli \
```

Comment:
as the es keystoreclit is mentioning 'elasticsearch' (https://github.com/elastic/elasticsearch/blob/6.x/server/src/main/java/org/elasticsearch/common/settings/KeyStoreCli.java#L32), we should use our own one.

Reply:
I have a PR for that https://github.com/crate/elasticsearch/pull/147